build.gradle

```diff
@@ -189,6 +190,13 @@ allprojects {
   apply plugin: 'java'
 
   dependencies {
+
+    // Add lombok
+    annotationProcessor 'org.mapstruct:mapstruct-processor:1.3.1.Final', 'org.projectlombok:lombok:1.18.16', 'org.projectlombok:lombok-mapstruct-binding:0.2.0'
+    annotationProcessor group: 'org.projectlombok', name: 'lombok', version: lombok
```

Comment:
There seems to be two instances of the lombok annotation processor defined on these lines.  Are both needed?  If only one is needed, should it be the version defined in the lombok variable or 1.18.16?

Reply:
removed duplicate one

Reply:
Confirmed duplicate annotation processor entry has been removed.